src/gleam/list.gleam

```diff
@@ -1452,3 +1452,49 @@ pub fn last(list: List(a)) -> Result(a, Nil) {
   list
   |> reduce(fn(elem, _) { elem })
 }
+
+/// Return unique combinations of elements in the list
+///
+/// ## Examples
+///
+/// ```
+/// > combinations_by([1, 2, 3], 2)
+/// [[1, 2], [1, 3], [2, 3]]
+///
+///  > combinations_by([1, 2, 3, 4], 3)
+///  [[1, 2, 3], [1, 2, 4], [1, 3, 4], [2, 3, 4]]
+/// ```
+///
+pub fn combinations_by(items: List(a), n: Int) -> List(List(a)) {
```

Comment:
What does the by mean? Would there also be a "combinations" function?

Reply:
I changed this to plain `combinations` and added `by` as named argument.

```
combinations([1,2,3], by: 2)
```

Reply:
Very nice